polaris-react/src/components/Tooltip/Tooltip.tsx

```diff
@@ -7,6 +7,10 @@ import {useToggle} from '../../utilities/use-toggle';
 
 import {TooltipOverlay, TooltipOverlayProps} from './components';
 
+export type Width = 'default' | 'wide';
+export type Padding = 'default' | 'loose';
+export type BorderRadius = 'default' | 'slight';
```

Comment:
```suggestion
export type BorderRadius = 'default' | 'large';
```

Since the "slight" is an increase in border-radius from the default, it might be clearer named something like "large"